header-gossip-network-test-vectors.md

```diff
@@ -0,0 +1,38 @@
+# Header Gossip Network Test Vectors
+
+This document contains test vectors for the Header Gossip Network to help implementations conform to the specification.
+
+## Header Accumulator Encodings
+
+These test vectors specify the proper encoding of the header accumulator in certain well defined scenarios
+
+### Epoch 0 Accumulator
+
+#### Accumulator after genesis block
+
+This test vector represents the input parameters and the expected output for the current epoch accumulator after adding
+the genesis block from Ethereum Mainnet.
+
+##### Input Parameters
+```
+genesis block hash = 0x88e96d4537bea4d9c05d12549907b32561d3bf31f45aae734cdc119f13406cb6
```

Comment:
This is block 1, not the genesis block. 
It does raise the question, should the accumulator not start from the genesis block?

Reply:
I would agree and just included the wrong block in error. Will update.

Reply:
Have added the genesis block and rerun the accumulator based on this.